packages/integrations/mdx/src/remark-images-to-component.ts

```diff
@@ -20,7 +20,7 @@ export function remarkImageToComponent() {
 			// checking if an image should be imported or not
 			if (file.data.imagePaths?.has(node.url)) {
 				let importName = importedImages.get(node.url);
-
+				const prefixedNodeUrl = (node.url.includes('/') ? '' : './') + node.url;
```

Comment:
This doesn't work when accessing files relatively like `foo/img.png`. I think the correct fix is to do a secondary `this.resolve` at https://github.com/withastro/astro/blob/799f6f3f29a3ef4f76347870a209ffa89651adfa/packages/astro/src/vite-plugin-markdown/index.ts#L76-L84 if it doesn't start with a `/`, which is kinda unfortunate for perf, but I suppose it's not something users would hit often.

Reply:
How about if it does not start with a / or ./, we append a `./` by default? That doesn't require use to resolve I think?

Reply:
Oh wait in case of `foo/img.png` it looks for it in the present directory of the markdown?

Reply:
If it's `foo/img.png`, it's synonymous to `./foo/img.png`. We can't add `./` by default because users can do `@/img.png` and expect the alias to resolve correctly.

Reply:
Right that's another case I didn't think of. Alright, I'll push the resolve thing. Makes sense. Thank you.

Reply:
@bluwy,

This works as expected with prefixedNodeUrl as verified using tests.